src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -22,52 +25,72 @@ public static void insertMetaData(String form, int id, int benutzerID, int grupp
         insertOrUpdate(sql);
     }
 
-    public static List<Map<String, String>> selectAttribute(String table, String attribute, Map<String, String> andConditions) throws Exception {
+    public static List<Map<String, String>> getAttribute(String table, String attribute, Map<String, String> andConditions) throws Exception {
```

Comment:
Another comment related to the signature: If we only select a single attribute, i don't think it makes much sense to use `List<Map<String,String>>` as a result type. A simple `List<String>` should be enough. I think  a `List<Map<String,String>>` only makes sense if we select multiple attributes at the same time.

Reply:
done: look at commit: changed Map<String, String> to Map

i can not use List<String> cause in do save i work with List<Map> attributes before change it was List<Map<String, String> attributes and i changed it back to List<Map> like it was original. and changed the function name now to public static List<Map> getAttribute(String table, String attribute, Map andConditions) throws Exception